lib/datasource/remote/api/eos_repository.dart

```diff
@@ -28,12 +28,22 @@ abstract class EosRepository {
   String actionNameAcceptnew = 'acceptnew';
   String actionNameRecover = 'recover';
   String actionNameUnplant = 'unplant';
+  String proposalActionNameDelegate = 'delegate';
+  String proposalActionNameUndelegate = 'undelegate';
 
   // Authorizations
   String permissionActive = 'active';
   String permissionOwner = 'owner';
   String permissionApplication = 'application';
 
+  // Voice scopes
```

Comment:
I think we should comment that found.seeds == campaign

Reply:
yes done